packages/admin/admin-color-picker/src/ColorPicker.tsx

```diff
@@ -57,7 +57,6 @@ export interface ColorPickerProps extends Omit<InputWithPopperProps, "children"
     fullWidth?: boolean;
     startAdornment?: InputBaseProps["startAdornment"];
     endAdornment?: InputBaseProps["endAdornment"];
-    invalidIndicatorCharacter?: string;
```

Comment:
@jamesricky any idea why we introduced this in https://github.com/vivid-planet/comet/pull/597. 

As far as I can see this prop is not used.

Reply:
I think so too 👍